File: data/src/main/java/com/clloret/days/data/remote/entities/ApiTag.java
package com.clloret.days.data.remote.entities;

import android.support.annotation.NonNull;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ApiTag {

  @Expose(serialize = false)
  private transient String id;

  @SerializedName("Name")
  private String name;

  // empty constructor needed by the Airtable library
  @SuppressWarnings("unused")
  public ApiTag() {

  }

  public ApiTag(String id) {

    this.id = id;
  }

  @NonNull
  public String getId() {

    return id;
  }

  public void setId(String id) {

    this.id = id;
  }

  public String getName() {

    return name;
  }

  public void setName(String name) {

    this.name = name;
  }
}

File: data/src/main/java/com/clloret/days/data/remote/entities/ApiTag.kt
package com.clloret.days.data.remote.entities

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ApiTag {
  @Expose(serialize = false)
  @Transient
  var id: String? = null

  @Expose(serialize = false)
  @Transient
  var createdTime: String? = null

  @SerializedName("Name")
  var name: String = ""

  // empty constructor needed by the Airtable library
  constructor()
  constructor(id: String?, name: String) {
    this.id = id
    this.name = name
  }

}